src/ui/public/chrome/directives/kbn_loading_indicator.js

```diff
@@ -0,0 +1,17 @@
+import UiModules from 'ui/modules';
+import angular from 'angular';
+
+const spinnerTemplate = '<div class="spinner" ng-show="chrome.httpActive.length"></div>';
+
+UiModules
+.get('ui/kibana')
+.directive('kbnLoadingIndicator', function ($compile) {
+  return {
+    restrict: 'AC',
```

Comment:
Can you get rid of the `C` here? This is only used as an attribute and only in one place right?